packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -373,6 +378,73 @@ export function useDeliveryPromise({
     []
   )
 
+  function getBadgesLabel(value: string) {
```

Comment:
idea: add the `availability` as arg to this function and return also the labels for unavailability